share/completions/just.fish

```diff
@@ -1,4 +1,4 @@
 function _justfile_targets
-    just --summary 2>/dev/null | string split ' '
+    just -l | tail -n +2 | sed -e 's/^[[:blank:]]*//' -e 's/[[:blank:]]*#/\t/' -e 's/[[:blank:]]*[*+][^[:blank:]]*//'
```

Comment:
Have you considered `string trim` and `string replace`?

We're trying to get away from `sed`, it's got a bunch of terrible implementations in use (I believe some don't support `\t` on the replacement-side).

Reply:
yes, this should do it then ;)